### ISSUE TITLE
Small "Doc Bug" in "circuitpython.readthedocs.io/en/latest/shared-bindings/countio/index.html"

### ISSUE THREAD

**Issue by ThomasAtBBTF** [adafruit/circuitpython#4352]
The small example in the section above contains an small error:
```python
import countio
import time
from board import *

# error "board not defined" here -> pin_counter = countio.Counter(board.D1)
pin_counter = countio.Counter(D1) # this is the fix
#reset the count after 100 counts
while True:
    if pin_counter.count == 100:
        pin_counter.reset()
    print(pin_counter.count)
```
Thomas

**Comment by ladyada** [comment]
hi please submit a PR with a documentation fix!

**Comment by dhalbert** [comment]
I was fixing another doc bug so I included this in PR #4359.
